horovod/spark/common/util.py

```diff
@@ -763,12 +763,17 @@ def to_list(var, length):
     return var
 
 
-def _get_assigned_gpu_or_default(default):
+def _get_assigned_gpu_or_default(default, hvd=None):
```

Comment:
I think `default` is not (and never was) a good naming here. It should be `local_rank`. With `local_size` as the second argument, `hvd` is not needed and the method becomes more readable. This also means the method has to be renamed.

Reply:
For the function `_get_assigned_gpu_or_default` itself, `default` is a passed in parameter, and `default` refers to `local_rank` is actually reflected in `spark/keras/remote.py` and other `remote.py` files. If we must change the parameter name here, maybe we'd better change the function name too. Then it will seem to be too much. So, I suggest we keep the `default` for now. If we really want to do the refactor, maybe we should do it in another PR.

Reply:
Thanks for suggestions. Updated code accordingly.